backend/views/syson-diagram-general-view/src/main/java/org/eclipse/syson/diagram/general/view/nodes/EmptyDiagramNodeDescriptionProvider.java

```diff
@@ -177,4 +160,31 @@ private NodeTool addExistingElementsTool() {
                 .body(addExistingelements.build())
                 .build();
     }
+    
+    private NodeToolSection[] createToolSections(IViewDiagramElementFinder cache) {
+        var sections = new ArrayList<NodeToolSection>();
+        
+        GeneralViewDiagramDescriptionProvider.TOOL_SECTIONS.forEach((sectionName, elements) -> {
+            NodeToolSectionBuilder sectionBuilder = this.diagramBuilderHelper.newNodeToolSection()
+                    .name(sectionName)
+                    .nodeTools(this.createElementsOfToolSection(cache, elements));
+            sections.add(sectionBuilder.build());
+        });
+        
+        sections.add(addElementsToolSection(cache));
```

Comment:
Could you add `this.` in front of every class variables/methods please? Could you also check everywhere else you have modified code in this PR?

Reply:
Done